lib/assert.js

```diff
@@ -299,7 +295,7 @@ function expectedException(actual, expected) {
     return false;
   }
 
-  if (Object.prototype.toString.call(expected) == '[object RegExp]') {
+  if (Object.prototype.toString.call(expected) === '[object RegExp]') {
```

Comment:
(And I see that function is replaced with the one from `internal/util` in https://github.com/nodejs/node/pull/11128.)